beacon_node/lighthouse_network/Cargo.toml

```diff
@@ -64,7 +64,7 @@ quick-protobuf-codec = "0.3"
 [dependencies.libp2p]
 version = "0.53"
 default-features = false
-features = ["identify", "yamux", "noise", "dns", "tcp", "tokio", "plaintext", "secp256k1", "macros", "ecdsa", "metrics", "quic"]
```

Comment:
We don't want gossipsub here anymore right?

Reply:
yeah, thanks age updated!